ansible/group_vars/all.yml

```diff
@@ -42,3 +42,31 @@ rhacm_redisgraph_memory_limit: "3Gi"
 hub_cluster_kubeconfig: /home/stack/ocp_clusters/vlan608/auth/kubeconfig
 managed_cluster_name: shiftstack0000
 managed_cluster_kubeconfig: "/home/stack/ocp_clusters/{{managed_cluster_name}}/auth/kubeconfig"
+
+################################################################################
+# Local-storage vars:
+################################################################################
+local_storage_instances:
+  key: beta.kubernetes.io/instance-type
+  value: nvme-8
```

Comment:
Is this expectation that the user labels the appropriate nodes with the this label prior to running the playbook?

Reply:
The instance type label is already slapped on to the shiftstack clusters I have been building.  In this case since our instance type also indicates passthrough nvme, we can leverage that label to search for the available nodes with local nvme.  The user could put a different label on storage nodes and select off of that too if they wanted to.